.travis.yml

```diff
@@ -32,7 +35,7 @@ matrix:
 
 jobs:
   allow_failures:
-    - script: yarn test:saucelabs --database-firestore-only
+    - script: ./scripts/saucelabs/database_firestore_test.sh
```

Comment:
maybe name this "run_flaky_tests.sh" or something so we can add/remove tests without renaming the script?

Reply:
Done. I agree. This script (and flakiness measurement in the future) may not be limited to only database and firestore tests.